### ISSUE TITLE
Support for setting custom VM parameters

### ISSUE THREAD

**Issue by YanChii** [erigones/esdc-ce#392]
I really don't know if this is already implemented or not. But it would be nice to support setting any exotic vmadm parameter during VM creation.
For example customer runs a database inside an LX zone and needs to set `max_lwps` for DB to work correctly.

This can be just API feature and we can have a whitelist of such vmadm parameters which can be set like this.

**Comment by dn0** [comment]
I suggest we implement this in a way that will allow only SuperAdmins to edit the whole VM's json configuration.

**Comment by YanChii** [comment]
I agree. And maybe these additional parameters can be set also in templates.
